core/backend/src/Entity.ts

```diff
@@ -86,6 +86,9 @@ export class Entity {
 
   /** @internal */
   public static get protectedOperations(): string[] { return []; }
+
+  /** get */
```

Comment:
you should provide some docs for this method

Reply:
added proper docs to this function